engine/edge.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Docker CE Edge releases
+description: Information and changelogs for Docker CE Edge releasers
+---
+
+## Overview
+
+Docker CE Edge releases are monthly releases cut straight from the `master`
+branch of the Docker repository. These releases allow you to try the latest
+features of Docker in non-production environments. They are effectively previews
+of the next Docker CE Stable release. Unless problems are found, these features
+will be available in the next releases of Docker CE and Docker EE.
```

Comment:
> "Unless problems are found, these features will be available in the next releases of Docker CE and Docker EE"

That would be a really rare exception; once a feature is "in" it's "in". Features in "Edge" releases are _not_ "experimental" features; they are _actual_ features.

All releases in the new version scheme / release cadence (including Edge) are _time_ driven, but not "unattended snapshots" (like a nightly build)